package.json

```diff
@@ -0,0 +1,736 @@
+    "tapable": "^2.2.1",
+    "tar": "^6.1.12",
+    "tar-fs": "^2.1.1",
+    "tar-stream": "^2.2.0",
+    "terser": "^5.16.1",
+    "terser-webpack-plugin": "^5.3.6",
+    "text-table": "^0.2.0",
+    "through": "^2.3.8",
+    "thunky": "^1.1.0",
+    "to-fast-properties": "^2.0.0",
+    "to-readable-stream": "^1.0.0",
+    "to-regex-range": "^5.0.1",
+    "toidentifier": "^1.0.1",
+    "token-types": "^4.2.1",
+    "tough-cookie": "^4.1.2",
+    "tr46": "^3.0.0",
+    "trim-newlines": "^3.0.1",
+    "trough": "^1.0.5",
+    "tsconfig-paths": "^3.14.1",
+    "tslib": "^2.4.1",
+    "tsutils": "^3.21.0",
+    "type-check": "^0.4.0",
+    "type-fest": "^0.18.1",
+    "type-is": "^1.6.18",
+    "typedarray-to-buffer": "^3.1.5",
+    "typescript": "^4.9.3",
+    "unbox-primitive": "^1.0.2",
+    "unbzip2-stream": "^1.4.3",
+    "unified": "^9.2.2",
+    "unique-filename": "^2.0.1",
+    "unique-slug": "^3.0.0",
+    "unique-string": "^2.0.0",
+    "unist-util-find-all-after": "^3.0.2",
+    "unist-util-is": "^4.1.0",
+    "unist-util-stringify-position": "^2.0.3",
+    "universalify": "^2.0.0",
+    "unpipe": "^1.0.0",
+    "update-browserslist-db": "^1.0.10",
+    "update-notifier": "^5.1.0",
+    "uri-js": "^4.4.1",
+    "url-parse": "^1.5.10",
+    "url-parse-lax": "^3.0.0",
+    "util-deprecate": "^1.0.2",
+    "utila": "^0.4.0",
+    "utils-merge": "^1.0.1",
+    "uuid": "^8.3.2",
+    "v8-compile-cache": "^2.3.0",
+    "validate-npm-package-license": "^3.0.4",
+    "validate-npm-package-name": "^4.0.0",
+    "vary": "^1.1.2",
+    "vfile": "^4.2.1",
+    "vfile-message": "^2.0.4",
+    "vm2": "^3.9.12",
+    "w3c-xmlserializer": "^4.0.0",
+    "watchpack": "^2.4.0",
+    "wbuf": "^1.7.3",
+    "wcwidth": "^1.0.1",
+    "webidl-conversions": "^7.0.0",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.1",
+    "webpack-dev-middleware": "^5.3.3",
+    "webpack-dev-server": "^4.11.1",
+    "webpack-merge": "^5.8.0",
+    "webpack-sources": "^3.2.3",
+    "websocket-driver": "^0.7.4",
+    "websocket-extensions": "^0.1.4",
+    "whatwg-encoding": "^2.0.0",
+    "whatwg-mimetype": "^3.0.0",
+    "whatwg-url": "^11.0.0",
+    "which": "^2.0.2",
+    "which-boxed-primitive": "^1.0.2",
+    "wide-align": "^1.1.5",
+    "widest-line": "^3.1.0",
+    "wildcard": "^2.0.0",
+    "word-wrap": "^1.2.3",
+    "wrap-ansi": "^7.0.0",
+    "wrappy": "^1.0.2",
+    "write-file-atomic": "^3.0.3",
+    "ws": "^8.12.0",
+    "xdg-basedir": "^4.0.0",
+    "xml-name-validator": "^4.0.0",
+    "xmlchars": "^2.2.0",
+    "xregexp": "^2.0.0",
+    "yallist": "^4.0.0",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "yauzl": "^2.10.0",
+    "zwitch": "^1.0.5"
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "html-webpack-plugin": "^5.5.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- [ ] according to the requirements Webpack dev server should be set up and running with no errors.